spec/features/lifehacks/02_user_views_details_of_lifehack_spec.rb

```diff
@@ -4,11 +4,12 @@
 
 feature "user sees the details of a lifehack" do
   let (:user) { FactoryGirl.create(:user) }
-  let!(:lifehack) { FactoryGirl.create(:lifehack) }
-  let!(:lifehack2) { FactoryGirl.create(:lifehack) }
+  let(:lifehack) { FactoryGirl.create(:lifehack) }
+  let(:lifehack2) { FactoryGirl.create(:lifehack) }
```

Comment:
You should be using `let!` here like you were before, instead of needing to call the name of the variable in your test unnecessarily!